tests/conftest.py

```diff
@@ -0,0 +1,81 @@
+import os
+import sys
+
+import mock
+import click
+import pytest
+
+import mock_tables.dbconnector
+
+import sonic_device_util
+from swsssdk import ConfigDBConnector
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+sys.path.insert(0, modules_path)
+
+generated_services_list = [
+    'ntp-config.service',
+    'warmboot-finalizer.service',
+    'watchdog-control.service',
+    'rsyslog-config.service',
+    'interfaces-config.service',
+    'hostcfgd.service',
+    'hostname-config.service',
+    'topology.service',
+    'updategraph.service',
+    'config-setup.service',
+    'caclmgrd.service',
+    'procdockerstatsd.service',
+    'pcie-check.service',
+    'process-reboot-cause.service',
+    'dhcp_relay.service',
+    'snmp.service',
+    'sflow.service',
+    'bgp.service',
+    'telemetry.service',
+    'swss.service',
+    'database.service',
+    'database.service',
+    'lldp.service',
+    'lldp.service',
+    'pmon.service',
+    'radv.service',
+    'mgmt-framework.service',
+    'nat.service',
+    'teamd.service',
+    'syncd.service',
+    'snmp.timer',
+    'telemetry.timer']
+
+
+def _dummy_run_command(command, display_cmd=False, return_cmd=False):
+    if display_cmd == True:
+        click.echo(click.style("Running command: ", fg='cyan') + click.style(command, fg='green'))
```

Comment:
Suggest simply having the dummy command always print the command. Then this will be useful to other subcommands which don't set `display_cmd=True`.

```
def _dummy_run_command(command, display_cmd=False, return_cmd=False):
    click.echo(command)
```

Reply:
I am not really to make this change as some output are explicitly hidden in the show command and user won't see all commands. if we show all the commands, then it will not be the same as what user see in the real box.

Reply:
Yes, my point was with this method, we can verify that the all the underlying commands are run correctly, even the ones that are hidden.

Reply:
it is better to change the show command its self to have a verbose mode where we can check all commands. But I think part of the unit test to verify we are seeing exactly the same as user see so that we won't have surprise in terms of what user sees.

Reply:
we can have a verbose option in load_minigraph to show all commands, and we can validate that in the unit test. but I think that is going to be a separate pr.

Reply:
Point taken. All commands *should* already support a `verbose` flag which gets passed to `run_command`. I guess for those commands we can test the function with the flag set to both True and False.